presto-main-base/src/main/java/com/facebook/presto/execution/SafeEventLoopGroup.java

```diff
@@ -86,6 +92,13 @@ public void execute(Runnable task, Consumer<Throwable> failureHandler)
                         failureHandler.accept(t);
                     }
                 }
+                finally {
+                    long cpuTimeInNanos = THREAD_MX_BEAN.getCurrentThreadCpuTime() - start;
+                    statsTracker.recordEventLoopMethodExecutionCpuTime(cpuTimeInNanos);
+                    if (cpuTimeInNanos > slowMethodThresholdOnEventLoopInNanos) {
+                        log.warn("Slow method execution on event loop: {} took {} nanoseconds", methodSignature, cpuTimeInNanos);
```

Comment:
If we are always adding the runtime stat... then i think we can skip the threshold config and logging part. We dont check the logs that often. We can instead monitor the runtime stats . So lets delete the threshold config , and logging logic.